tests/proxy/mocks/ads/recommendations/campaign-budget.json

```diff
@@ -0,0 +1,84 @@
+{
+	"results": [
+        {
+            "recommendation": {
+                "type": "CAMPAIGN_BUDGET",
+                "resource_name": "customers/123/recommendations/1",
+				"campaign_budget_recommendation": {
```

Comment:
In `src/API/Google/BudgetRecommendations.php`, I noticed the condition handling cases where `getCampaignBudgetRecommendation()` may return empty:
[https://github.com/woocommerce/google-listings-and-ads/blob/bbb59d6aab89a85d2c6d38965b27a3692b566cd7/src/API/Google/BudgetRecommendations.php#L100-L103](https://github.com/woocommerce/google-listings-and-ads/blob/bbb59d6aab89a85d2c6d38965b27a3692b566cd7/src/API/Google/BudgetRecommendations.php#L100-L103)

Do we need to verify whether this is working as expected or if it could potentially surface a bug?

cc. @joemcgill